README.adoc

```diff
@@ -36,10 +36,18 @@ Explore how to externalize configuration using MicroProfile Config and configure
 // =================================================================================================
 
 == What you'll learn
-The https://openliberty.io/guides/microprofile-config.html[MicroProfile Config guide] teaches you how to externalize configuration from your application. It may not be immediately clear why this is beneficial to you when developing your microservices. You will learn how to externalize your configuration, and provide the configuration values by creating environment variables. By using MicroProfile Config to handle your configuration, it provides useful annotations that you may use to inject configured values into your code. These values can come from environment variables or elsewhere.
+The https://openliberty.io/guides/microprofile-config.html[Configuring microservices guide] teaches you how to externalize configuration from your application. It may not be immediately clear why this is beneficial to you when developing your microservices, but {kube} provides some tools that allow you to conveniently configure your microservices with a setup such as the one described. You will learn how to externalize your configuration, and provide the configuration values by creating environment variables. By using MicroProfile Config to handle your configuration, it provides useful annotations that you may use to inject configured values into your code. These values can come from environment variables or elsewhere.
 
 Furthermore, you'll learn how to set these environment variables using ConfigMaps and Secrets. These resources are provided by {kube} that act as a data source for your environment variables. By creating a ConfigMap or Secret, they can be used to set environment variables for any number of containers as you see fit.
 
+== Prerequesites
+
+Some basic knowledge of {kube} is assumed for this guide, and you should have a {kube} cluster available for your use. All the prerequesite knowledge is covered in the https://openliberty.io/guides/kubernetes.html[Deploying microservices to Kubernetes] guide. If you need a {kube} cluster to work with, we recommend using Docker Desktop or Minikube. Alternatively, you can provision a cluster from a cloud provider of your choice or use an existing cluster. If you choose to go with one of the local single-node clusters, you can find information on setting up Docker Desktop or Minikube at these URLs:
```

Comment:
"we recommend" => do not use first person, rephrase the sentence to use second person. Also don't need the word "recommend", it offers options which can be unclear to user (this is the feedback from Laura). Just provide a way.

Reply:
prerequisite *

Reply:
We should be specific in terms of what you should have for which platform.